src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,28 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger =
+            (Logger) LogManager.getLogger(AuthenticationServiceImpl.class);
```

Comment:
Casting `LogManager.getLogger(AuthenticationServiceImpl.class)` to `Logger` is unnecessary. You can directly assign it to `logger` without casting.